streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ApplicationService.java

```diff
@@ -120,4 +122,6 @@ List<Application> getByTeamIdAndExecutionModes(
   List<String> historyUploadJars();
 
   String k8sStartLog(Long id, Integer offset, Integer limit) throws Exception;
+
+  void changeUser(Long userId, Long targetUserId);
```

Comment:
It is recommended to change the method name to `changeOwnership`

Reply:
okay